test/ml/test_ml.py

```diff
@@ -102,7 +106,7 @@ def test_pca_robustness(self):
             coord = coord.round(5)  # round values to 5 digits to account for numeric differences across machines
             coord.sort_values(by='algorithm', ignore_index=True, inplace=True)
 
-            assert coord.equals(expected)
+            assert coord.equals(expected) or coord.equals(expected_other)
```

Comment:
Is there 2 possible outputs?

Reply:
I believe the answer is yes? It should be valid to flip the solution.